ci/travis-after-success.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/sh
 set -xe
 
+# Submit code coverage report
+curl -s https://codecov.io/bash | bash
```

Comment:
> With some trial and error we could probably reproduce it but it seems somewhat fragile to me with all hard-coded urls. What do you think about simply storing the whole script inside ocean repo? It doesn't download / execute any other scripts internally AFAICS so that would work as sort of approved pinning.

Yeah, thinking about it, probably running the script with `docker run` (but without sharing the whole build dir, only the reports, and all the env vars with the container) it should be safe and convenient enough, even if we keep using the remote script (although I would download it from GitHub, not the codecov.io URL).